stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -542,6 +542,11 @@ public func expectCrashLater() {
   _seenExpectCrash = true
 }
 
+public func expectCrash(_ disaster: () -> Void) {
```

Comment:
A minor thing, but: maybe `expectCrash(invoking:)` or `expectCrash(executing:)`. It does not matter with the trailing-closure syntax, but when you pass an existing function there, it would look nicer. What do you think?

Reply:
I completely agree with you. I will do in shortly.